bucket/open-tv.json

```
@@ -0,0 +1,35 @@
+{
+    "version": "1.3.0",
+    "description": "Ultra-fast, simple and open-source IPTV app",
+    "homepage": "https://github.com/Fredolx/open-tv",
+    "license": "GPL-2.0-only",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/fredolx/open-tv/releases/download/v1.3.0/Open.TV_1.3.0_x64_en-US.msi",
+            "hash": "700EE0A9D963227D7803D439AED1C37094E901A4BA772A401A5A259FFEB4C21E"
+        }
+    },
+    "extract_dir": "PFiles\\Open TV",
+    "bin": "open_tv.exe",
+    "shortcuts": [
+        [
+            "open_tv.exe",
+            "Open TV"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/fredolx/open-tv"
+    },
```

```suggestion
    "checkver": "github",
```

Enough since GH link already used as homepage.